app/models/listing_unit.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: listing_units
+#
+#  id                  :integer          not null, primary key
+#  unit_type           :string(32)       not null
+#  translation_key     :string(64)
+#  transaction_type_id :integer
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#
+
+class ListingUnit < ActiveRecord::Base
+  attr_accessible(
+    :transaction_type_id,
```

Comment:
transaction_type_id, for now. We don't have yet listing_shape model or table. Would be very confusing to name it listing_shape_id, when it is actually transaction_type_id